src/store/constants.js

```diff
@@ -23,4 +23,4 @@ export const UNIFIED_ACCOUNT_ID = 0
 export const UNIFIED_INBOX_ID = 'unified'
 export const PRIORITY_INBOX_ID = 'priority'
 export const PAGE_SIZE = 20
-export const UNDO_DELAY = 7 * 1000
+export const UNDO_DELAY = 10 * 1000
```

Comment:
yes but no. This now *matches* the value of the dialogs package, but there is no guarantee that they will stay in sync. See https://github.com/nextcloud/nextcloud-dialogs/pull/210. The constant is exposed. Let's get the value from the lib. If the value ever changes there, it wouldn't break our code.